build/bootstrap_mcs.sh

```diff
@@ -0,0 +1,183 @@
+#!/bin/sh
+
+# This script compiles/installs MCS from scratch 
+# The script presumes that server's source code is two directories
+# above the MCS engine source.
+# This script is to be run under root.
+
+DISTRO=$1
+MDB_BUILD_TYPE=$2
+MCS_SCRIPTS_REPO_PREFIX=/data/cs-docker-tools
+MDB_SOURCE_PATH=/data/mdb-server
+MCS_SOURCE_PATH=$MDB_SOURCE_PATH/storage/columnstore
+MCS_CONFIG_DIR=/etc/columnstore
+# Needs systemd to be installed obviously.
+# Feel free to ask in MariaDB Zulip how to bootstrap MCS in containers or other systemd-free environments.
+systemctl stop mariadb-columnstore
+MDB_GIT_URL=https://github.com/MariaDB/server.git
+MCS_GIT_URL=https://github.com/mariadb-corporation/mariadb-columnstore-engine.git
+MDB_GIT_TAG=10.5
```

Comment:
can be parsed from .drone.jsonnet inside 
```
local server_ref_map = {
  develop: '10.7',
  'develop-6': '10.6-enterprise',
  'develop-5': '10.5',
  '**': '10.7',
}
```

Reply:
I doubt it can given the private enterprise repo mentioned at least.